boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1278,6 +1278,16 @@ LoadAndBootWindowsCommon(
     WinLdrpDumpArcDisks(LoaderBlockVA);
 #endif
 
+    /* Show a notice to user about debugger connection */
+    if (strstr(LoaderBlockVA->LoadOptions, "DEBUG"))
```

Comment:
The problem is that this also matches the `NODEBUG` boot option.
Could you test whether the following works? :
```suggestion
    if (!strstr(LoaderBlockVA->LoadOptions, "CRASHDEBUG") &&
        !strstr(LoaderBlockVA->LoadOptions, "NODEBUG") &&
        strstr(LoaderBlockVA->LoadOptions, "DEBUG"))
```

Reply:
Oh you're right
I forgot to test it without debug option.
Is it efficient to call "strstr" multiple times?